### ISSUE TITLE
Suggestion to add warning in the docs when using nautilus

### ISSUE THREAD

**Issue by camipacifici** [ACCarnall/bagpipes#84]
Hi Adam,
After a few hiccups, I managed to fit my spectrum and it works beautifully with nautilus.
I would like to suggest to add a word of caution when using nautilus, though. When I started experiment with the fits, they would never converge and I had no idea why. The priors were sensible and even the simplest spectrum would not work. Then I switched to verbose=True and realized that the runner was stuck in a try/except statement. Basically, it kept resuming the previous run (I didn't know it would do that), but in the meantime I had changed the spectrum limits and resolution and the priors, so I guess the runner was very confused and didn't know what to do.
Maybe you can add to the docs something explicit about resuming previous runs similar to what is in the nautilus docs here: https://nautilus-sampler.readthedocs.io/en/stable/guides/checkpointing.html#checkpointing
Thank you again for your help!!

**Comment by johannesulf** [comment]
I'm happy to hear nautilus worked for you. And you're right that one should not resume a fit using a checkpoint file from a different fit. That will actually confuse both nautilus and MultiNest. I don't think it's necessarily specific to nautilus unless there are differences in the way bagpipes implements these two samplers regarding checkpointing.

**Comment by johannesulf** [comment]
I had a look at the code and, I believe, MultiNest would be affected in the same way. So if the documentation is updated, the statement about resuming from previous runs can be a general statement, I believe.

**Comment by camipacifici** [comment]
Makes sense. Thank you for clarifying this. I am not a frequent user of either of the two.

**Comment by ACCarnall** [comment]
Hmm, yes both multinest and nautilus will resume from previous partially completed runs if they can. Multinest warns about this  whether verbose is set to True or False, whereas it looks like nautilus never explicitly warns the user about this - it's just implicit if you have verbose set to True as it stars at non-zero values for calls etc in the printed output. Might be worth thinking about I suppose Johannes? It's a fair point though that even with multinest printing a warning I've known this to confuse people before, so a warning in the bagpipes docs probably would be a good idea.
